src/app/portlets/content-types/fields/content-type-fields-properties-form/field-properties/dynamic-field-property-directive/dynamic-field-property.directive.ts

```diff
@@ -40,13 +37,15 @@ export class DynamicFieldPropertyDirective implements OnChanges {
         const componentRef: ComponentRef<any> = this.viewContainerRef.createComponent(
             componentFactory
         );
-
         componentRef.instance.property = {
             field: this.field,
             name: this.propertyName,
             value: this.field[this.propertyName]
         };
 
         componentRef.instance.group = this.group;
+        componentRef.instance.helpText = this.fieldPropertyService.getFieldType(
+            this.field.clazz
+        ).helpText;
```

Comment:
This should go into the values-property.component, but to do this change we need move  the getFieldType from FieldPropertyService to FieldService.

Also I think that the method has more sense into FieldService, maybe we should move the 'getProperties' method too